src/server/pfs/server/testing/server_test.go

```diff
@@ -5029,6 +5029,15 @@ func TestPFS(suite *testing.T) {
 				commit := commits[i]
 				commits = append(commits[:i], commits[i+1:]...)
 				require.NoError(t, env.PachClient.SquashCommitSet(commit.ID))
+				// some output branches might have had new commits added either because their old heads
+				// were squashed or because the head of a branch they are provenant on was squashed
+				for _, branch := range outputBranches {
```

Comment:
This change isn't passing, but this change does:
```
ris, err := env.PachClient.ListRepo()
require.NoError(t, err)
for _, ri := range ris {
	bis, err := env.PachClient.ListBranch(ri.Repo.Name)
	require.NoError(t, err)
	for _, bi := range bis {
		branch := bi.Branch
		info, err := env.PachClient.InspectCommit(branch.Repo.Name, branch.Name, "")
		require.NoError(t, err)
		if info.Finishing == nil {
			require.NoError(t, finishCommit(env.PachClient, branch.Repo.Name, branch.Name, ""))
		}
	}
}
```
I'm not exactly sure why, but we should go ahead and get this in so we can have a green CI.